src/pages/ServiceList/ServiceListComponent.tsx

```diff
@@ -1,30 +1,64 @@
 import * as React from 'react';
-import { ListView, ListViewItem, ListViewIcon, Button } from 'patternfly-react';
+import { ListView, ListViewItem, ListViewIcon } from 'patternfly-react';
 import { Link } from 'react-router-dom';
 
 class ServiceListComponent extends React.Component {
   render() {
     return (
       <div>
         <ListView>
-          <ListViewItem
-            key="Product Page"
-            leftContent={<ListViewIcon name="plane" />}
-            heading={
-              <span>
-                Product Page
-                <small>Feb 23, 2015 12:32 am</small>
-              </span>
-            }
-            actions={
-              <div>
-                <Link to={'/namespaces/istio-system/services/ProductPage'}>
-                  <Button>Details</Button>
-                </Link>
-              </div>
-            }
-            description={<span />}
-          />
+          <Link to={'/namespaces/istio-system/services/ProductPage'} style={{ color: 'black' }}>
```

Comment:
please use lowercase 'productpage' here as this is what the backend knows.
Same for ratings and reviews further down.

Reply:
By covention K8S world works with lowercase names
https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
So I agree we should move these even in a mock.
Also, UI speaking Kubernetes console also follows this
https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/